tests/src/python/test_db_manager_gpkg.py

```diff
@@ -444,6 +444,32 @@ def testAllGeometryTypes(self):
 
         connection.remove()
 
+    def testAmphibiousMode(self,):
+        connectionName = 'geopack1'
+        plugin = createDbPlugin('gpkg')
+        uri = QgsDataSourceUri()
+        test_gpkg = os.path.join(self.basetestpath, 'test_amphibiousGpkg.gpkg')
```

Comment:
can't we just use one of the existing .gpkg files like tests/testdata/points_gpkg.gpkg ?

Reply:
Possibly but since all the other tests in this file use the same method and since this does not depend on a common source I felt that this was more appropriate. I'm open to change if needed.

Reply:
I believe other tests need to create a file to test a particular aspect. Here you can use any .gpkg file and use any ST_ function from Spatialite to demonstrate it's working as expected. The less code of lines, the better

Reply:
Strangely the test fail, maybe its because the polygons are using curved lines.

Reply:
Yeah curved features could not be processed, test is fine now.